jira_scraper/common/constants.py

```diff
@@ -3,9 +3,10 @@
 COLLECTION_NAME = "all-jira-tickets"
 DEFAULT_EMBEDDING_MODEL = "BAAI/bge-large-en-v1.5"
 DEFAULT_JIRA_URL = "https://issues.redhat.com"
-DEFAULT_JIRA_PROJECTS = [
-    "OSP", "RHOSINFRA", "OSPCIX", "RHOSBUGS",
-    "OSPK8", "RHOSPRIO", "OSPRH"
-]
+DEFAULT_JIRA_PROJECTS = {
+    "OSP": "New",
```

Comment:
that data structure is not intuitive, let's keep it a list of projects. Also, I wouldn't constraint ourselves to a state of the tickets, let's get them all for now, which is I thought what we said.

Reply:
Done